packages/st2bundle/install_rhel6.md

```diff
@@ -0,0 +1,48 @@
+## Install IUS repo
+sudo yum install -y https://dl.iuscommunity.org/pub/ius/stable/Redhat/6/x86_64/ius-release-1.0-14.ius.el6.noarch.rpm
+
+# Install mongodb
+sudo yum install -y mongodb mongodb-server
+
+# Install rabbitmq
+sudo rpm --import https://www.rabbitmq.com/rabbitmq-signing-key-public.asc
+curl -sS -k -o /tmp/rabbitmq-server.rpm https://www.rabbitmq.com/releases/rabbitmq-server/v3.3.5/rabbitmq-server-3.3.5-1.noarch.rpm
+sudo yum localinstall -y /tmp/rabbitmq-server.rpm
+
+## Install st2
+sudo touch /etc/yum.repos.d/StackStorm-el6_staging-unstable.repo
+
+Add following lines to /etc/yum.repos.d/StackStorm-el6_staging-unstable.repo
+[StackStorm-el6_staging-unstable]
+name=StackStorm-el6_staging-unstable
+baseurl=https://dl.bintray.com/stackstorm/el6_staging/unstable
+enabled=1
+gpgcheck=0
+
+sudo yum install st2bundle
+
+
+## Start mongodb
+sudo chkconfig mongod on
+sudo service mongod start
+
+## Start rabbitmq
+sudo chkconfig rabbitmq-server on
+sudo service rabbitmq-server start
+
+## Start st2 services
+sudo st2ctl start
+
+## Register content
+st2-register-content --config-file=/etc/st2/st2.conf --register-all
+
+## Setup auth user
+sudo yum install httpd-tools
+sudo htpasswd -cs /etc/st2/htpasswd admin
+
+## Install git (Fix st2bundle to require git.)
+sudo yum install git-all
```

Comment:
This step is not required anymore, should be deleted. I've fixed it https://github.com/StackStorm/st2-packages/blob/master/packages/st2bundle/rpm/st2bundle.spec#L7-L11